src/byro/office/signals.py

```diff
@@ -45,3 +45,19 @@
 to import is in the `upload_file` form field) and should return a Response
 object.
 """
+member_dashboard_tile = django.dispatch.Signal()
+"""
+This signal allows you to add tiles to the member's dashboard.
+Receives None as argument, must return either None or a dict:
```

Comment:
Larger discussion for this kind of signal: I always have the inner urge to make them return a list instead of a dict, in case a plugin wants to return more than one. Or make it optionally return a dict OR a list (slightly complicates the handling code).
(Trade-off: For a limited value of "more than one" the plugin can just register the signal multiple times. This does not work if the maximum number of tiles to show isn't known beforehand, and also it's kind of ugly.)

Reply:
YES! I only added it as an object for consistency's sake.